docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Dockerfile for awesomeWM.
```

Comment:
It is already in a subdir?!

Inititally I had multiple, e.g. `.dep` for the base dependencies - might come back to this..